x-pack/test/api_integration/deployment_agnostic/apis/observability/alerting/custom_threshold/documents_count_fired.ts

```diff
@@ -5,33 +5,35 @@
  * 2.0.
  */
 
+import { omit } from 'lodash';
+import expect from '@kbn/expect';
 import { cleanup, generate, Dataset, PartialConfig } from '@kbn/data-forge';
 import { Aggregators } from '@kbn/observability-plugin/common/custom_threshold_rule/types';
 import { FIRED_ACTIONS_ID } from '@kbn/observability-plugin/server/lib/rules/custom_threshold/constants';
-import expect from '@kbn/expect';
 import { OBSERVABILITY_THRESHOLD_RULE_TYPE_ID } from '@kbn/rule-data-utils';
 import { parseSearchParams } from '@kbn/share-plugin/common/url_service';
-import { omit } from 'lodash';
 import { COMPARATORS } from '@kbn/alerting-comparators';
 import { kbnTestConfig } from '@kbn/test';
-import { FtrProviderContext } from '../../../ftr_provider_context';
+import type { InternalRequestHeader, RoleCredentials } from '@kbn/ftr-common-functional-services';
+import { DeploymentAgnosticFtrProviderContext } from '../../../../ftr_provider_context';
 import { ISO_DATE_REGEX } from './constants';
-import { ActionDocument, LogsExplorerLocatorParsedParams } from './typings';
-import type { InternalRequestHeader, RoleCredentials } from '../../../../shared/services';
+import { ActionDocument, LogsExplorerLocatorParsedParams } from './types';
 
-export default function ({ getService }: FtrProviderContext) {
+export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
   const esClient = getService('es');
   const supertest = getService('supertest');
```

Comment:
Shouldn't `supertestWithoutAuth` be used here?

Reply:
Good catch, I'll update it.

Reply:
Thanks for the clarification; I updated all the `supertest` usage in [a140e02](https://github.com/elastic/kibana/pull/198691/commits/a140e0260823ff8dd463c05969f632a320562647)